### ISSUE TITLE
packer build panic if option -on-error=ask and -force are used all together

### ISSUE THREAD

**Issue by chambodn** [hashicorp/packer#10241]
#### Overview of the Issue

It sounds like the usage of both -on-error=ask and -force options make packer crash.

#### Reproduction Steps
Execute the following command:
```bash
$ packer build -force -on-error=ask build.json
```
### Packer version
Packer v1.6.5

### Simplified Packer Buildfile
```json
{
    "variables": {
        "project_id": "{{ env `PROJECT_ID` }}",
        "image_name": "image-test-01",
        "account_file": "{{ env `GCP_ACCOUNT_FILE` }}",
        "zone": "europe-west1-b"
    },
    "builders": [
        {
            "type": "googlecompute",
            "account_file": "{{ user `account_file` }}",
            "project_id": "{{ user `project_id` }}",
            "image_name": "{{ user `image_name` }}",
            "source_image_family": "ubuntu-2004-lts",
            "disk_size": "50",
            "disk_type": "pd-ssd",
            "state_timeout": "15m",
            "machine_type": "n1-standard-2",
            "preemptible": true,
            "use_internal_ip": false,
            "communicator": "ssh",
            "zone": "{{ user `zone` }}",
            "ssh_username": "packer"
        }
    ]
}
```

### Operating system and Environment details
MacOS Catalina Version 10.15.7
go1.15.3 darwin amd64

### Log Fragments and crash.log files
Crash log [here](https://gist.github.com/chambodn/53eed88a8ae1c564076b51af97131ba5)

**Comment by nywilken** [comment]
Thanks for reaching out we will take a look as soon as we get a chance.

**Comment by nywilken** [comment]
Hello again, thanks for opening up this issue. It looks like just using `-on-error=ask` or `-on-error=abort` will cause the builder to crash. I was able to reproduce the issue and make a potential fix in #10247. There are some test binaries available [here](https://app.circleci.com/pipelines/github/hashicorp/packer/7942/workflows/fb3d26dc-f092-46c7-882a-3ef62d5d8288/jobs/92013/artifacts) if you would like to pull them down and give the patch a try.

**Comment by chambodn** [comment]
Hello @nywilken I confirm the binaries you produced fix the issue.
Thanks a lot !
